packages/api/src/controllers/v2/deliberations/_id/final_surveys.rs

```diff
@@ -108,6 +108,33 @@ impl DeliberationFinalSurveyController {
             Some(Authorization::Bearer { ref claims }) => AppClaims(claims).get_user_id(),
             _ => 0,
         };
+
+        let mut is_member = false;
+
+        if user_id != 0 {
+            let user = User::query_builder()
+                .id_equals(user_id)
+                .query()
+                .map(User::from)
+                .fetch_one(&self.pool)
+                .await?;
+
+            let email = user.email;
+
+            let emails: Vec<String> = Deliberation::query_builder()
+                .id_equals(deliberation_id)
+                .query()
+                .map(Deliberation::from)
+                .fetch_one(&self.pool)
+                .await?
+                .roles
+                .iter()
+                .map(|v| v.email.clone())
+                .collect();
+
+            is_member = emails.contains(&email);
```

Comment:
`roles` 가 아니라 `emails` 아닐까요?
Survey 참가자 인지가 중요할 것 같아서요.

```suggestion
let is_memeber = DeliberationPanelEmail::query_builder()
            .deliberation_id_equals(deliberation_id)
            .email_equals(email)
            .query()
            .map(Into::into)
            .fetch_optional(&self.pool)
            .await?.is_some();
```

Reply:
잘못 구현된 부분인데 이메일이어야할거 같네요. 수정했습니다!